src/schema/objects/metadata.yaml

```diff
@@ -976,6 +1033,53 @@ EthicsApprovals:
   type: array
   items:
     type: string
+EventIdentifier:
+  name: EventIdentifier
+  display_name: Event Identifier
+  description: |
+    The message sent to the eye-tracker which disambiguates the belonging
+    of the sample to a certain event, condition or group.
+    This can be a unique trial identifier, or in the case of
+    continuous presentation (for example video) the annotated event of interest.
+  type: string
+EyeCameraSettings:
+  name: EyeCameraSettings
+  display_name: Eye Camera Settings
+  description: |
+    A field to store any settings that influence the resolution
+    and quality of the eye imagery.
+    Autowhitebalance? Changes in sharpness?
+  type: object
+  items:
+    type: object
+EyeTrackerDistance:
```

Comment:
I would add these in addition to the simpler `EyeTrackerDistance`, noting that it is recommended to have these better metadata values. I'd suggest also:

```JSON
"EyeToCameraXYZ": (0.05, 0.30, 0.40),
"EyeToScreenTopLeftXYZ": (0.15, 0.1, 0.5),
```

Reply:
So people can choose if they want to use `EyeTrackerDistance` or `EyeToCameraXYZ` ? Asking for the converter...

Reply:
The thing is also that so far in the spec `ScreenDistance` is a **required** metadata whereas `EyeTrackerDistance` is **recommended**. So if we want to have both 
```
"EyetrackingGeometry" : {
  "EyeToCameraXYZ": [0.05, 0.30, 0.40],
  "EyeToScreenTopLeftXYZ": [0.15, 0.1, 0.5],
}
``` 
(which needs to be **required** because it contains the required ScreenDistance) as well as the choice to simply use `ScreenDistance` and `EyeTrackerDistance` instead we should talk about if we want to make `EyeTrackerDistance` required as well, to ensure some equality between the two choices. But might not be necessary, just a question!

Reply:
The schema was updated: https://github.com/bids-standard/bids-specification/pull/1128/commits/6e787b6d6d129383038d4d4a87683285760c01f2

There are no `<name>XYZ` keys now, and we don't have dictionaries (on top of my head).

Item 3 refers to the existing BIDS specs, under the events section. Rather than make the events file REQUIRED, I'd suggest that in case both are defined, eye tracking should always be referred to the eyetracking-specific metadata.

Reply:
ok thanks!